Example/EditorDemo/WPViewController.m

```diff
@@ -39,15 +37,15 @@ - (void)customizeAppearance
     // by any of the Noto varients.  Size is arbitrary.
     // See: https://github.com/wordpress-mobile/WordPress-Shared-iOS/issues/79
     // Remove this when #79 is resolved.
-    [WPFontManager notoRegularFontOfSize:16.0];
-    [WPFontManager loadNotoFontFamily];
-
-    self.placeholderColor = [WPStyleGuide grey];
-    self.editorView.sourceViewTitleField.font = [WPFontManager notoBoldFontOfSize:24.0];
-    self.editorView.sourceContentDividerView.backgroundColor = [WPStyleGuide greyLighten30];
-    [self.toolbarView setBorderColor:[WPStyleGuide greyLighten10]];
-    [self.toolbarView setItemTintColor: [WPStyleGuide greyLighten10]];
-    [self.toolbarView setSelectedItemTintColor: [WPStyleGuide baseDarkerBlue]];
+//    [WPFontManager notoRegularFontOfSize:16.0];
+//    [WPFontManager loadNotoFontFamily];
+
+//    self.placeholderColor = [WPStyleGuide grey];
+//    self.editorView.sourceViewTitleField.font = [WPFontManager notoBoldFontOfSize:24.0];
+//    self.editorView.sourceContentDividerView.backgroundColor = [WPStyleGuide greyLighten30];
+//    [self.toolbarView setBorderColor:[WPStyleGuide greyLighten10]];
+//    [self.toolbarView setItemTintColor: [WPStyleGuide greyLighten10]];
+//    [self.toolbarView setSelectedItemTintColor: [WPStyleGuide baseDarkerBlue]];
```

Comment:
Maybe add a descriptive `TODO:` or other comment here reminding us about the shared pod dependency being removed?